src/components/Button.js

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 function Button(props) {
-  const { name, color, wide } = props;
+  const {
+    name, color, wide, onClick,
+  } = props;
```

Comment:
You could destructure on the argument: `function Button({ name, color, wide, onClick }) {`

Reply:
Nice, much more cleaner